crates/nu-data/src/value.rs

```diff
@@ -161,6 +175,15 @@ pub fn compute_values(
                         }
                         Ok(x % bigdecimal::BigDecimal::from(y.clone()))
                     }
+                    Operator::Pow => {
+                        let xp = bigdecimal::ToPrimitive::to_f64(x).unwrap_or(0.0);
```

Comment:
operations on bigint and bigdecimal shouldn't leave the big form and go back to floats and ints.

Reply:
commented out for now. i left a question on the bigdecimal repo about pow-ing bigdecimals.